meta-dts-distro/recipes-dts/dts-scripts/dts-scripts_git.bb

```diff
@@ -0,0 +1,44 @@
+SUMMARY = "Dasharo Tools Suite scripts."
```

Comment:
The name of the recipe can stay as `dts` as previously, IMHO.

Reply:
IMHO, this is not an image recipe, but a recipe that provides scripts for that image, so name `dts-scripts` is more clear.

Reply:
I am not saying it is an image recipe. But OK, let it stay this way :+1: